Microsoft.Toolkit.Services/Services/OneDrive/OneDriveStorageFolder.cs

```diff
@@ -0,0 +1,408 @@
+// ******************************************************************
+// Copyright (c) Microsoft. All rights reserved.
+// This code is licensed under the MIT License (MIT).
+// THE CODE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+// IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+// TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH
+// THE CODE OR THE USE OR OTHER DEALINGS IN THE CODE.
+// ******************************************************************
+
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Graph;
+using Microsoft.Toolkit.Services.OneDrive.Platform;
+using static Microsoft.Toolkit.Services.MicrosoftGraph.MicrosoftGraphEnums;
+
+namespace Microsoft.Toolkit.Services.OneDrive
+{
+    /// <summary>
+    /// OneDriveStorageFolder Type
+    /// </summary>
+    public class OneDriveStorageFolder : OneDriveStorageItem
```

Comment:
Any reason why the UWP version of these classes(OneDriveStorageFile, OneDriveStorageFolder, OneDriveStoreItem, etc) are not extending these classes?

Reply:
Discussed. UWP versions have coupling to OneDrive SDK, which we remove for .NET Standard version.  Will mark OneDrive SDK dependencies obsolete.